external/CMakeLists.txt

```diff
@@ -14,6 +14,26 @@
 #
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+include(FetchContent)
+
+# Enable C and C++ language extensions which we otherwise disable, as
+# external projects are entitled to assume CMake defaults.
+set(CMAKE_C_EXTENSIONS ON)
+set(CMAKE_CXX_EXTENSIONS ON)
+
+# OpenCL Headers are used throughout the project, so make them available
```

Comment:
I get why you've gone this way, but it would be good to get rid of the libimg dependency and keep it in the source/cl area at some point.
Could add a comment to say that's it's the libimg dependency that requires it rather than throughout.
Just a suggestion though - feel free to ignore.

Reply:
There's more than just that, there's also `modules/mux/targets/host/extension/example/main.c` which is outside of `source/cl`. But I'll edit the comment to focus mainly on libimg.

Reply:
Edited, does it look okay to you like this?

Reply:
sure